index.html

```diff
@@ -1,65 +1,207 @@
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+
+            <div class="card grid-item">
+                <div class="card-image">
+                    <img src="./images/Snapshoot Portfolio 2.png">
+                </div>
+                <div class="card-info">
+                    <p class="project-name">Tonic</p>
+                    <div class="project-info">
+                        <div class="company">CANOPY</div>
+                        <div class="dot"></div>
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+
+            <div class="card grid-item card-2">
+                <div class="card-image">
+                    <img src="./images/Snapshoot Portfolio 3.png">
+                </div>
+                <div class="card-info">
+                    <p class="project-name">Multi-Post Stories</p>
+                    <div class="project-info">
+                        <div class="company">CANOPY</div>
+                        <div class="dot"></div>
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="about-section">
+            <div class="about-info">
+                <p class="title">About Myself</p>
+                <p class="content">
+                    Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
                 <p class="connect">LET'S CONNECT</p>
```

Comment:
- [ ] Please use `hx` tags like `h3` or `h4` instead of `p`, because adopting the use of **hx** tags over **p** tags for headings ensures better accessibility, improved SEO, consistent styling, and enhanced user experience on your website. I kindly request you to please incorporate this practice into your HTML coding for future projects and updates.